src/compile/compile.ts

```diff
@@ -82,7 +82,7 @@ function assembleTopLevelModel(model: Model, topLevelProperties: TopLevelPropert
   // move width and height signals with values to top level
   layoutSignals = layoutSignals.filter(signal => {
     if ((signal.name === 'width' || signal.name === 'height') && signal.value !== undefined) {
-      topLevelProperties[signal.name] = signal.value;
+      topLevelProperties[signal.name] = +signal.value;
```

Comment:
Shoudn't signal values already be a number?

Reply:
Can't they be strings or bools?

Reply:
By type, yes.  By how we initialize width and height signals, should be no (unless we have a bug).

Reply:
Right, this is just a type thing. I guess we could say `as number`